diff -U 5 a/.bashrc b/.bashrc
--- a/.bashrc	2024-06-13 00:00:00.000000000 +0000
+++ b/.bashrc	2024-06-13 00:00:00.000000000 +0000
@@ -28,11 +28,11 @@
     debian_chroot=$(cat /etc/debian_chroot)
 fi
 
 # set a fancy prompt (non-color, unless we know we "want" color)
 case "$TERM" in
-    xterm-color) color_prompt=yes;;
+    xterm-color|*-256color) color_prompt=yes;;
 esac
 
 # uncomment for a colored prompt, if the terminal has the capability; turned
 # off by default to not distract the user: the focus in a terminal window
 # should be on the output of commands, not on the prompt
